lib/dynamoid/components.rb

```diff
@@ -23,7 +23,7 @@ module Components
     include ActiveModel::Naming
     include ActiveModel::Observing if defined?(ActiveModel::Observing)
     include ActiveModel::Serializers::JSON
-    include ActiveModel::Serializers::Xml
+    #include ActiveModel::Serializers::Xml
```

Comment:
I understand this was removed from Rails 5, but commenting it out is not appropriate.  probably ` include ActiveModel:: Serializers if defined?(ActiveModel:: Serializers)` is the easiest fix here, as done above for ActiveModel::Observing